_posts/2022-07-08-dev-sprint-leaders.md

```diff
@@ -0,0 +1,55 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-07-08 13:00:00
+layout: post
+title: "Lead or organize a Development Sprint at DCUS 2022!"
+---
+
+The last two days of DjangoCon US *(this year, October 20-21)* are devoted to Development Sprints. This blog post is targeted at current open source maintainers and contributors who are considering organizing or leading a Development Sprint at DCUS 2022. As such, I won't be defining "Development Sprint". That's for another blog post.
```

Comment:
>As such, I won't be defining "Development Sprint". That's for another blog post. 

I'd rephrase to "A more specific definition of "Development Sprint" will come in a later post.